src/main.js

```diff
@@ -198,3 +199,12 @@ function saveNewPoster() {
     savedPosters.push(currentPoster);
   }
 }
+
+function deletePoster() {
+  for (var i = 0; i < savedPosters.length; i++) {
+    if (`${savedPosters[i].id}` === event.target.id) {
```

Comment:
I don't think you need to interpolate here, I think you can just write ` if (savedPosters[i].id === event.target.id)`
Nice use of splice!